ironfish-cli/src/commands/wallet/multisig/create-signature-share.ts

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { Flags } from '@oclif/core'
+import { IronfishCommand } from '../../../command'
+import { RemoteFlags } from '../../../flags'
+
+export class CreateSignatureShareCommand extends IronfishCommand {
+  static description = `Creates a signature share for a participant for a given transaction`
+  static hidden = true
+
+  static flags = {
+    ...RemoteFlags,
+    account: Flags.string({
+      char: 'f',
+      description: 'The account from which the signature share will be created',
+      required: false,
+    }),
+    unsignedTransaction: Flags.string({
+      char: 'u',
+      description: 'The unsigned transaction for which the signature share will be created',
+      required: true,
+    }),
+    signingPackage: Flags.string({
+      char: 's',
+      description: 'The signing package for which the signature share will be created',
+      required: true,
+    }),
+  }
+
+  async start(): Promise<void> {
+    const { flags } = await this.parse(CreateSignatureShareCommand)
+    const signingPackage = flags.signingPackage
```

Comment:
nit - could we pass this directly from `flags` below like we do with other args?